doc/user/content/guides/cdc-mysql.md

```diff
@@ -35,6 +35,13 @@ Before deploying a Debezium connector, you need to ensure that the upstream data
 
     FLUSH PRIVILEGES;
     ```
+1. Materialize currently does not support truncation for upstream sources. {{% gh 7276 %}}. If you have truncation enabled on MySQL, you can disable it by changing the `sql-mode`:
```

Comment:
We shouldn't prompt users to disable truncation; it's just not supported downstream.